pkgs/tools/misc/optinix/default.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  fetchFromGitLab,
+  buildGoModule,
+  installShellFiles,
+}:
+buildGoModule rec {
+  pname = "optinix";
+  version = "0.1.1";
+
+  src = fetchFromGitLab {
+    owner = "hmajid2301";
+    repo = "optinix";
+    rev = "v${version}";
+    hash = "sha256-bRHesc03jExIL29BCP93cMbx+BOT4sHCu58JjpmRaeA=";
+  };
+
+  vendorHash = "sha256-uSFEhRWvJ83RGpekPJL9MOYJy2NfgVdZxuaNUMq3VaE=";
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+    installShellCompletion --cmd optinix \
+      --bash <($out/bin/optinix completion bash) \
+      --fish <($out/bin/optinix completion fish) \
+      --zsh <($out/bin/optinix completion zsh)
+  '';
+
+  meta = with lib; {
+    description = "Tool for searching options in Nix";
+    homepage = "https://gitlab.com/hmajid2301/optinix";
+    license = licenses.mit;
+    maintainers = with maintainers; [ hmajid2301 ];
+    mainProgram = "optinix";
+    platforms = platforms.linux ++ platforms.darwin;
+  };
```

Obviously, my remark was todo something similar to:

```suggestion
  meta = {
    description = "Tool for searching options in Nix";
    homepage = "https://gitlab.com/hmajid2301/optinix";
    license = lib.licenses.mit;
    maintainers = with lib.maintainers; [ hmajid2301 ];
    mainProgram = "optinix";
    platforms = with lib.platforms; [ linux ] ++ [ darwin ];
  };
```

---

ahh of course :see_no_evil: